Apsitvarkom.UnitTests/Models/Validation/CleaningEventUpdateValidatorTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Apsitvarkom.Models.Public;
+using FluentValidation;
+
+namespace Apsitvarkom.UnitTests.Models.Validation;
+
+public class CleaningEventUpdateValidatorTests
```

Comment:
looking at our code we typically have the name `'PublicModel'ValidationTests` here